### DISCUSSION TITLE
Make clickAction a computable method instead of just a $clickAction style property

### DISCUSSION THREAD

**Discussion by Zephni** [laravel/nova-issues#6111] [Ideas & Feature Requests]
I tend to set up a Trait that I can add to my Nova Resources with some defaults such as:

    public static function tableStyle()
    {
        return 'tight';
    }
    
And redirectAfterCreate, redirectAfterUpdate etc, which all work fine. But it seems that...
    
    public static function clickAction()
    {
        return 'edit';
    }

...doesn't work this way, and it's not possible to set the property itself within the trait as this will cause an error and for good reason. Is there any chance we can allow clickAction() to be used as a method instead of the generic property so it can be overriden or computed with a method?

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
